### PR TITLE
Explain usage of `flask openapi` command more clearly

### LINKED ISSUE #63: `flask openapi` command not available
Environment:

- Python version: 3.9.16
- Operating system: MacOS 13.3
- Flask version: 2.2.3
- flask-openapi3 version: 2.3.2

The `flask openapi` command mentioned in the documentation, https://luolingchun.github.io/flask-openapi3/Tutorial/Specification/#command-flask-openapi, is not available.

### PR DESCRIPTION
**Pull request by candleindark**
Checklist:

- [x] Run `pytest tests` and no failed.
- [x] Run `flake8 flask_openapi3 tests examples` and no failed.
- [x] Run `mkdocs serve` and no failed.
- [x] Run `mypy flask_openapi3` and no failed.

I have provided a more clear explanation of the usage of the `flask openapi` command in this PR. This PR should close #63.

### CODE REVIEW

File: docs/Tutorial/Specification.md [hunk 1]
```diff
@@ -6,16 +6,25 @@ If you need the complete Specification, go to http://127.0.0.1:5000/openapi/open
...
+The `flask openapi` command will export the OpenAPI Specification to console when you execute the command,
 
 ```
-flask openapi
+flask --app IMPORT openapi
 ```
+where `IMPORT` is the Flask application, in our case an OpenAPI application, to loan.
+For example, if your OpenAPI application is `app` defined in `hello.py`, 
+as in the example in [Quickstart](https://luolingchun.github.io/flask-openapi3/Quickstart/#rest-api), the command is
+`flask --app hello.py:app openapi `.
```
Comment (luolingchun) [inline docs/Tutorial/Specification.md:17]: `hello:app` is better than `hello.py:app`

File: docs/Tutorial/Specification.md [hunk 2]
```diff
@@ -6,16 +6,25 @@ If you need the complete Specification, go to http://127.0.0.1:5000/openapi/open
...
 
-Execute `flask openapi --help` for more information:
+
+Execute `flask --app IMPORT openapi --help` for more information about the command:
+
+Again, assuming your OpenAPI application is `app` defined in `hello.py`,
 
 ```
-flask openapi --help
+flask --app hello.py:app openapi --help
```
Comment (luolingchun) [inline docs/Tutorial/Specification.md:27]: `hello:app` is better than `hello.py:app`

File: docs/Tutorial/Specification.md (hunk 1 above)
Comment (candleindark) [reply docs/Tutorial/Specification.md:17]: Yeah, go ahead and make the change you suggested. 谢谢。

File: docs/Tutorial/Specification.md (hunk 2 above)
Comment (candleindark) [reply docs/Tutorial/Specification.md:27]: Yeah, go ahead and make the change you suggested. 谢谢。

**Pull request merged**